builtin/providers/aws/utils.go

```diff
@@ -0,0 +1,13 @@
+package aws
```

Comment:
Yeah, I was looking for examples in other packages on what to name this file. In the `aws` package, there's an `auth_helper.go`. In other packages there are `utils.go`, just not sure which is the best fit for this use case.